gm4_fulcio_shamir/data/gm4_fulcio_shamir/loot_tables/band.json

```diff
@@ -0,0 +1,36 @@
+{
+    "pools": [
+        {
+            "rolls": 1,
+            "entries": [
+                {
+                    "type": "minecraft:loot_table",
+                    "name": "gm4_metallurgy:barimium_band",
+                    "functions": [
+                        {
+                            "function": "minecraft:set_nbt",
+                            "tag": "{gm4_metallurgy:{stored_shamir:\"fulcio\",custom_model_data:115}}"
```

Comment:
This should be `CustomModelData` (`custom_model_data` is what it's called in resource packs)

Reply:
Also, the custom model data tag is stored inside the GM4 tag - I might've overlooked something here

Reply:
Ok, apparently this is something the module uses to set the tag - ignore me :p